<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortWidget</class>
 <widget class="QWidget" name="SortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1419</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SortWidget</string>
  </property>
  <widget class="QPushButton" name="btn5merge">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>归并排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>730</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2select">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择排序</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1071</width>
     <height>791</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1bubble">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>冒泡排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3insert">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>570</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>插入排序</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>300</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>添加数字：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>320</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4quick">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>570</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>快速排序</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>0</y>
     <width>321</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>随机生成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6radix">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>基数排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>690</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>停止动画</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>470</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
